.travis.yml

```diff
@@ -15,9 +15,28 @@
 #
 # Travis CI build file for Swift gRPC.
 
-os:
-  - linux
-  - osx
+matrix:
+  include:
+    - os: linux
+      osx_image: xcode10
+      env:
+        - SWIFT_VERSION=4.2
+        - MAJOR_VERSION=4.2
+    - os: osx
+      osx_image: xcode10
```

Comment:
Should we use Xcode 10.2 for the Swift 4.2 runs as well?

Reply:
Yep makes sense